src/core/lib/transport/static_metadata.cc

```diff
@@ -337,6 +337,324 @@ const grpc_slice grpc_static_slice_table[GRPC_STATIC_MDSTR_COUNT] = {
     {&grpc_static_metadata_refcounts[106], {{21, g_bytes + 1234}}},
 };
 
+grpc_mdelem grpc_static_mdelem_manifested[GRPC_STATIC_MDELEM_COUNT] = {
+    /* GRPC_MDELEM_AUTHORITY_EMPTY: ":authority": "" */
+    GRPC_MAKE_MDELEM(&grpc_static_mdelem_table[0].data(),
+                     GRPC_MDELEM_STORAGE_STATIC), /* GRPC_MDELEM_METHOD_GET:
+                                                     ":method": "GET" */
```

Comment:
The 2nd comment and afterwards, all have broken indentation. can you please fix that?

I believe they should be all on the same line. Or is it the gen script that's broken?

Reply:
gen script does things, then the clang_format_code does what it wants. I can play with it to see if I can get it better.

Reply:
Fixed by disabling clang formatting and rolling our own.